mkdocs.yml

```diff
@@ -232,6 +232,7 @@ nav:
                 - Interface between Graphene and hBN:                          tutorials/materials/specific/interface-2d-2d-graphene-boron-nitride.md
                 - Interface between Copper and SiO2 (Cristobalite):                  tutorials/materials/specific/interface-3d-3d-copper-silicon-dioxide.md
                 - Interface between Graphene and SiO2 (alpha-quartz):                tutorials/materials/specific/interface-2d-3d-graphene-silicon-dioxide.md
+                - High-k Metal Gate Stack:                                      tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md
```

Comment:
We should list the materials used: Si, SiO2, HfO2, TiN. Same for the name.

Reply:
`heterostructure-high-k-metal-gate-stack-silicon-silicon-dioxide-hafnium-dioxide-titanium-nitride.md` -- too long of a name?